migrationmanager/migrators/traces/migrations/000027_add_is_remote_span.down.sql

```diff
@@ -0,0 +1,47 @@
+DROP TABLE IF EXISTS signoz_traces.durationSortMV ON CLUSTER {{.SIGNOZ_CLUSTER}};
+
+ALTER TABLE signoz_traces.signoz_index_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}}
+    DROP COLUMN IF EXISTS isRemote;
+
+ALTER TABLE signoz_traces.distributed_signoz_index_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}}
+    DROP COLUMN IF EXISTS isRemote;
+
+ALTER TABLE signoz_traces.durationSort ON CLUSTER {{.SIGNOZ_CLUSTER}}
+    DROP COLUMN IF EXISTS isRemote;
+
+ALTER TABLE signoz_traces.distributed_durationSort ON CLUSTER {{.SIGNOZ_CLUSTER}}
+    DROP COLUMN IF EXISTS isRemote;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS signoz_traces.durationSortMV ON CLUSTER {{.SIGNOZ_CLUSTER}}
```

Comment:
drop MV

Reply:
dropping is the first statement in this down migration file. here we recreate the mv with old columns.

Reply:
oh right you are creating the old one